acr-repositories.yaml

```diff
@@ -154,3 +154,7 @@ rules:
     ruleName: ryuk-cache
     repoName: testcontainers/ryuk
     destinationRepo: imported/testcontainers/ryuk
+  alpine:
+    ruleName: alpine
+    repoName: alpine
```

Comment:
I think repoName needs "library" before it in this case.  From the README:

> Should always be format of publisher/image. If there is no publisher, please use "library".

Could you also add a new entry for alpine to the table in the README?